examples/dag-with-args-example/dag-job/dag_job.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from vdk.plugin.meta_jobs.meta_job_runner import MetaJobInput
+
+
+JOBS_RUN_ORDER = [
```

Comment:
I should probably have brought this up in one of the earlier dag PRs. 
But it is very weird that they are supposed to create list of dicts.
They are expected to know all the field names?
Can we not make this dict a class?
the class would be called DagNode.



Maybe this isn't the best place to bring this up. 

WDYT?

Reply:
You have a point here. This data modelling idea was brought up a while ago. There was even a task to do it but it was dropped primarily due to user feedback. Kinda surprisingly, the users turned out to be happy with the list of dicts. Therefore, we left it that way for now.

Reply:
If the customer wants it this way then I'm happy to mark it as resolved.